packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -955,7 +936,18 @@ export default class TradeStore extends BaseStore {
                             this.debouncedProposal();
                             this.clearLimitOrderBarriers();
                             this.pushPurchaseDataToGtm(contract_data);
-                            this.setContractPurchaseToastbox(response.buy);
+                            if (this.root_store.ui.is_mobile) {
+                                const shortcode = response.buy.shortcode;
+                                this.root_store.notifications.addTradeNotification({
+                                    buy_price: is_multiplier ? this.amount : response.buy.buy_price,
```

Comment:
```suggestion
                                    buy_price: is_multiplier ? this.amount : response?.buy?.buy_price,
```

Reply:
hi @ali-hosseini-deriv `response.buy` will be truthy when we reach this line :)